homeassistant/components/unifi/sensor.py

```diff
@@ -236,23 +250,31 @@ def async_client_wan_monitor_latency(
 
 @callback
 def async_device_latency_supported_fn(
```

Comment:
If you update the previous function to return None if there are no latency values, you can instead disregard this function and replace it with something like `async_client_wan_monitor_latency(...) is not None`

Reply:
Do you mean removing it or? 
Signatures are slightly different?

Reply:
Either removing it and using a lambda or something to go from hub to device or updating it to use the hub to get the device and then call the latency function.

Reply:
Exactly combine those functions into one that take which stat to look at and you will remove a lot of duplication

Reply:
I see, it's applied now. Thanks!